src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {

```

Comment:
According to the checklist, don't begin class implementation with an empty line. Please remove the empty line before the class declaration.